examples/simple/add_header.py

```diff
@@ -1,2 +1,6 @@
-def response(flow):
+from mitmproxy import http
+import typing
+
+
+def response(flow: http.HTTPFlow) -> typing.Any:
```

Comment:
`Any` has no significance here. since return type in` None` , it good to put `Any` than to ignore type.